assets/css/styleguide.scss

```diff
@@ -377,4 +377,12 @@ header[role="banner"] {
 
 .download-code {
   padding-top: 3em;
+}
+
+// Search bar grid --------- //
+
+.preview-search-bar {
+  .usa-grid-box {
+    padding: 0;
+  }
 }
```

Comment:
I don't think we need a new line if the new selector is directly following the parent selector with no properties or directives before it.